lib/pavilion/plugins/commands/clean.py

```diff
@@ -46,7 +53,8 @@ def run(self, pav_cfg, args):
                 cutoff_date = datetime.today() - timedelta(
                     weeks=int(args.older_than[0]))
             elif 'month' in args.older_than or 'months' in args.older_than:
-                cutoff_date = get_month_delta(int(args.older_than[0]))
```

Comment:
I forgot about this, but Kody's code is super wrong.

Here's a snippet I wrote to make sure:
```python
#!/usr/bin/python3

import argparse
import sys

p = argparse.ArgumentParser()
p.add_argument('--older-than', nargs='+', action='store')

args = p.parse_args()

print(args.older_than)
```

Reply:
As you'll see when you play with this, args.older_than is a list (because of nargs='+'), which allows for getting multiple values for this argument (which we don't want). Since the action is 'store' though, additional values given just overwrite the original list. 

This whole section doesn't work at all because when you look for 'days' in args.older_than, it will find stuff like '5 days' and will never match. If it did match, it would try to convert '5 days' into an int, which is a ValueError.